pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -906,6 +890,7 @@ func (c *Reconciler) createTaskRun(ctx context.Context, taskRunName string, para
 			Annotations:     combineTaskRunAndTaskSpecAnnotations(pr, rpt.PipelineTask),
 		},
 		Spec: v1beta1.TaskRunSpec{
+			Retries:            rpt.PipelineTask.Retries,
```

Comment:
Is this always defaulted to 1 when not set?

Reply:
This is default to 0, the TaskRun will always run for `tr.retries + 1` times.